sfdc/Module.md

```diff
@@ -332,13 +205,13 @@ REST API. Following are the remote functions available for retrieving SObject me
    <td>Query for actions displayed in the UI, given a user, a context, device format, and a record ID
    </td>
   </tr>
-</table>
+  </table>
+
 
-## Operations to get Organizational Data
-Apart from the main SObject related functions Ballerina Salesforce Connector facilitates users to get information about 
-their organization. Following are the remote functions available for retrieving organizational data. 
+- Operations to get organizational data  
```

Comment:
same here

Reply:
We decided to remove quick reference from module.md.